proxy/http/HttpSessionManager.cc

```diff
@@ -246,7 +239,7 @@ void
 HttpSessionManager::init()
 {
   m_g_pool = new ServerSessionPool;
-  eventProcessor.schedule_spawn(&initialize_thread_for_http_sessions, ET_NET);
+  eventProcessor.schedule_spawn([](EThread *thread){ thread->server_session_pool = new ServerSessionPool; }, ET_NET);
```

Comment:
Yeh that can get pulled out.  I think it was more a "nice thing" that is unlike the others .. and a proof of concept that it worked fine.

It's not critical to any functionality.